charts/stable/mealie/values.yaml

```diff
@@ -1,34 +1,28 @@
-#
 # IMPORTANT NOTE
 #
 # This chart inherits from our common library chart. You can check the default values/options here:
 # https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
 #
-
+---
```

Comment:
Can you please revert the header changes? We want to keep this the same throughout all charts.